brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-2066 - Fix: Button Hover stylings are not working in the editor.

### Description
Fix: Button Hover stylings are not working in the editor.


### Types of changes
Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?
Testing Evidence - [Screen Recording 18-11-2022 at 16.07](https://share.getcloudapp.com/2Nu6Jby7)

TC - 

- Activate the Astra theme and switch to next-release
- Drag and drop above blocks
- Apply background colour and text colour to the button
- update the page 
- check front-end and back-end
- switch to the branch mentioned in the PR
- regenerate assets and refresh the front-end
- Also, refresh the editor
- check hover color working properly or not

### Checklist:
